src/traits.rs

```diff
@@ -245,4 +270,18 @@ pub trait Signer {
         descriptors: Vec<ImportDescriptor>,
         wallet_name: String,
     ) -> impl Future<Output = ClientResult<Vec<ImportDescriptorResult>>> + Send;
+
+    fn wallet_process_psbt(
+        &self,
+        psbt: &str,
+        sign: Option<bool>,
+        sighashtype: Option<String>,
```

Comment:
Nit! No strings! We can use [`TapSighashType`](https://docs.rs/bitcoin/latest/bitcoin/sighash/enum.TapSighashType.html) and deal with the `0` that is not covered by [`EcdsaSighashType`](https://docs.rs/bitcoin/latest/bitcoin/sighash/enum.EcdsaSighashType.html) or create new `enum` in `types.rs` to deal with all sighashes from ECDSA and Taproot.